numpyro/distributions/directional.py

```diff
@@ -400,7 +400,7 @@ def norm_const(self):
 
         fs = (
             lbinoms.reshape(-1, 1)
-            + 2 * m * jnp.log(corr)
+            + m * jnp.log(corr**2)
```

Comment:
Nice catch! I guess you can clip by tiny here in case users set corr=0?

Reply:
Sure, that makes sense.